.chloggen/use-metric-key-with-aws-calculator.yaml

```diff
@@ -0,0 +1,11 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: awsemfexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Change to use metric key instead of metric and labels
```

Comment:
For what?  I can't understand what this change will do from this description.

Reply:
Duly noted. I have changed it to 
```
Instead of supply metric metadata and label for aws calculator, supply with aws metric key which consists of metric metadata and label
```

Reply:
Please help me review again @Aneurysm9